### PR TITLE
Development build instructions on x64-windows

### PR DESCRIPTION
**Pull request by coot**

### CODE REVIEW

File: doc/getting-started/building-the-node-on-windows.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+This is not a __PRODUCTION__ build!
+
+Building on `Windows` is a non trivial task.  There are various tools one needs
+to install and configure.  We recommend to use
+[chocolatey](https://chocolatey.org) (which provides the `choco` command) to
+install them.
+
+All instructrions that invoke `choco` can be done in a PowerShell with root
+previlages.  For development work we recommend installing and using `git-bash`
+(with `choco install git-bash`).
```
Comment (dorin100) [inline doc/getting-started/building-the-node-on-windows.md:13]: I am getting this error when executing `choco install git-bash`

`git-bash not installed. The package was not found with the source(s) listed.
 Source(s): 'https://chocolatey.org/api/v2/'
 NOTE: When you specify explicit sources, it overrides default sources.
If the package version is a prerelease and you didn't specify `--pre`,
 the package may not be found.
Please see https://chocolatey.org/docs/troubleshooting for more
 assistance.`

Comment (dorin100) [reply doc/getting-started/building-the-node-on-windows.md:13]: choco is installed correclty 
`PS C:\WINDOWS\system32> choco
Chocolatey v0.10.15`

File: doc/getting-started/building-the-node-on-windows.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+in PowerShell>
+
+We are about to transition to `ghc-8.10.2` but this version is broken on
+windows ([see](https://gitlab.haskell.org/ghc/ghc/-/issues/18550)), though
+a fix is rather easy.
+
+## Install pkg-config
+
+```PowerShell
+choco install pgkconfiglite
```
Comment (dorin100) [inline doc/getting-started/building-the-node-on-windows.md:31]: This also results in an error:
`PS C:\WINDOWS\system32> choco install pgkconfiglite
Chocolatey v0.10.15
Installing the following packages:
pgkconfiglite
By installing you accept licenses for the packages.
pgkconfiglite not installed. The package was not found with the source(s) listed.
 Source(s): 'https://chocolatey.org/api/v2/'
 NOTE: When you specify explicit sources, it overrides default sources.
If the package version is a prerelease and you didn't specify `--pre`,
 the package may not be found.
Please see https://chocolatey.org/docs/troubleshooting for more
 assistance.`

File: doc/getting-started/building-the-node-on-windows.md (hunk 1 above)
Comment (coot) [reply doc/getting-started/building-the-node-on-windows.md:13]: It's just `choco install git`

File: doc/getting-started/building-the-node-on-windows.md (hunk 2 above)
Comment (coot) [reply doc/getting-started/building-the-node-on-windows.md:31]: This works for me:
```
choco search pkgconfiglite
...
pkgconfiglite 0.28
1 packages found.
```

File: doc/getting-started/building-the-node-on-windows.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+This document describes how to build a __DEVELOPMENT__ version of `cardano-node`.
+This is not a __PRODUCTION__ build!
+
+Building on `Windows` is a non trivial task.  There are various tools one needs
+to install and configure.  We recommend to use
+[chocolatey](https://chocolatey.org) (which provides the `choco` command) to
+install them.
+
+All instructrions that invoke `choco` can be done in a PowerShell with root
+previlages.  For development work we recommend installing and using `git-bash`
```
Comment (newhoggy) [inline doc/getting-started/building-the-node-on-windows.md:12]: s/instructrions/instructions
s/previlages/privileges

File: doc/getting-started/building-the-node-on-windows.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+(with `choco install git-bash`).
+
+## Install GHC
+
+The recommended way is to
+
+```PowerShell
+choco install --version 8.6.5 ghc
```
Comment (newhoggy) [inline doc/getting-started/building-the-node-on-windows.md:20]: Needs to be 8.10.2 now.

Comment (coot) [reply doc/getting-started/building-the-node-on-windows.md:20]: Done

File: doc/getting-started/building-the-node-on-windows.md (hunk 3 above)
Comment (coot) [reply doc/getting-started/building-the-node-on-windows.md]: Done

**Comment by coot** [comment]
@newhoggy I included your suggestions and checked the spelling with a check-speller.

**Review by Jimbo4350 — APPROVED** [review]

**Review by newhoggy — APPROVED** [review]

**Comment by coot** [comment]
bors merge

**Pull request merged**